tests/node-redis-dockerfile

```diff
@@ -0,0 +1,21 @@
+# syntax=docker/dockerfile:1
```

Comment:
I did not know the convention of naming dockerfiles like `foo-dockerfile`.  Lets call it `node-redis.Dockerfile`. 

Also, why not add this file to `https://github.com/dragonflydb/node-redis`  (to dragonfly branch) and then we could introduce a CD the that creates this package (docker) for us to use. Then we could use the package without building it locally.

Reply:
We can push the image to ghr manually for now.

Reply:
Your decision. In the future I hope that we will not need to fork and we would be able to pull directly from the client lib github repo. We could try and see then if they will be ok with that. Also, having this lightwieght docker in the dragonflys repo feels right in terms of the responsibility of the developer to test and maintain the integration testing. Your call.